guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -7,6 +7,7 @@ You can use a configuration management tool to configure multiple hosts at once.
 .Prerequisites
 * The host must be using a supported operating system.
 For more information, see xref:supported-clients-in-registration_{context}[].
+* Either `curl` or `wget` must be installed on the host.
```

Comment:
I don't think it's needed in this procedure, I'd move it to Prerequisites in _Registering a host_.